Adicionado a resolução do exercício 02

samuelxcastro opened the pull request:
Adicionado a resolução do exercício 02.
Imprima apenas números pares.

samuelxcastro committed:
Adicionado a resolução do exercício 02

Adicionado a resolução do exercício 02.
Imprima apenas números pares.

devprocommunity commented:
Vamos mover o exercício para a pasta 002-NumerosPares

devprocommunity commented on Python/README.MD:
```diff
@@ -2,21 +2,24 @@
 
 Construir um algoritmo que **leia 2 números e efetue a adição**. **Caso** o valor somado seja **maior que 20**, este deverá ser apresentada **somando**-se a ele **mais 8**; **caso** o valor somado seja **menor ou igual a 20**, este deverá ser **apresentado subtraindo-se 5**.
```
podemos remover essa descrição e deixar só da linha 6.

devprocommunity commented on Python/README.MD:
```diff
@@ -2,21 +2,24 @@
->> Dicas para construir o código.
->> * Declare variáveis
+    # Exercício de Python 02
+Pegue o objeto lista = [10,11,12,13,17,21,99,33,47,14,18,24,50,60,70,80]; e eetorne apenas os valores pares.
```
corrigir palavra => retorne.

devprocommunity commented on Python/README.MD:
```diff
@@ -2,21 +2,24 @@
 
->> Dicas para construir o código.
->> * Declare variáveis
+    # Exercício de Python 02
```
Encoste o # titulo a esquerda, pra destacar seu título h1.

samuelxcastro committed:
Feito separação dos exercícios e Readme editado.

Feito separação dos exercícios, cada exercício em determinada pasta. Novas instruções no README.

devprocommunity commented:
Muito obrigado pela contribuição @samuelxcastro 🥇
